alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -278,6 +306,40 @@ def _create_session(self, concurrency):
         session.mount("http://", adapter)
         return session
 
+    def _load_file(self, worker_host, path):
+        try:
+            response = self.session.get(
+                self.LOAD_SUBMIT_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=path,
+                ),
+            )
+            response.raise_for_status()
+            while True:
+                response = self.session.get(
+                    self.LOAD_PROGRESS_URL_FORMAT.format(
+                        worker_host=worker_host,
+                        http_port=self.http_port,
+                        path=path,
+                    ),
+                )
+                response.raise_for_status()
+                content = response.content
+                if b"SUCCEEDED" in content:
+                    print(content.decode('utf-8'))
```

Comment:
Can we use log instead of print?
In addition, if the SUCCEEDED content only give information as load succeed, the load function return true always give such information, is it helpful to even log this message?

Reply:
Throughput will be logged.